src/browser-bookmarks/import-bookmarks.js

```diff
@@ -0,0 +1,41 @@
+import db from '../pouchdb'
+import { updatePageSearchIndex } from '../search/find-pages'
+import { getVisitItemsForHistoryItems, transformToPageDoc, transformToVisitDoc,
+   convertHistoryToPagesAndVisits } from '../browser-history/import-history'
+import { isWorthRemembering, generatePageDocId, generateVisitDocId,
+  visitKeyPrefix, convertVisitDocId } from '../activity-logger'
+
+    function onRejected(error) {
+      console.log(`An error: ${error}`);
+    }
+
+    function getBookmarkItems() {
+      var bookmarkTree = browser.bookmarks.getRecent(100000);
+      return bookmarkTree.then(bookmarkItems =>
+          bookmarkItems.filter(({url}) => isWorthRemembering({url})));
+    }
+
+    export default function importBookmarks() {
+      return getBookmarkItems().then(result =>
+      getVisitItemsForHistoryItems(result)).then(
+          res => {
+            var docs = convertHistoryToPagesAndVisits(res)
```

Comment:
Does this actually work without problems? You are passing bookmark items as if they were history items twice here (line 20&22). I suppose that it could work as they also just have a `url` and and `id` field, and that is all we use (I think?). We should probably make these functions from the history import a bit more generic to also work for bookmarks, e.g. `transformPageDoc` should probably set a different nonce depending on which import process uses it. Not sure whether we would want to deduplicate or link up bookmarks and history items when both are imported. Needs thought.